tutorials/intermediate/constrainedlayout_guide.py

```diff
@@ -430,6 +450,34 @@ def docomplicated(suptitle=None):
 example_plot(axs[0], fontsize=12)
 axs[1].set_position([0.2, 0.2, 0.4, 0.4])
 
+###############################################################################
+# .. _compressed_layout:
+#
+# Grids of fixed aspect-ratio Axes: "compressed" layout
+# =====================================================
+#
+# ``constrained_layout`` operates on the grid of "original" positions for
+# axes. However, when Axes have fixed aspect ratios, one side is usually made
+# shorter, and leaves large gaps in the shortened direction. In the following,
+# the Axes are square, but the figure quite wide so there is a horizontal gap:
+
+fig, axs = plt.subplots(2, 2, figsize=(5, 3),
+                        sharex=True, sharey=True, layout='constrained')
+for ax in axs.flat:
+    ax.imshow(arr)
+fig.suptitle("fixed-aspect plots, layout='constrained'")
+
+# One obvious way of fixing this is to make the figure size more square,
+# however, closing the gaps exactly requires trial and error.  For simple grids
+# of Axes we can use ``layout="compressed"`` to do the job for us:
```

Comment:
Needs the `####...` header if you want this to be text instead of a comment.

Reply:
That's really weird.  If I do this manually it works fine.  if I save it, it works fine.  However, I agree that the sphinx version does not work fine?